rackcity/models/network_port.py

```diff
@@ -25,6 +26,46 @@ class NetworkPort(models.Model):
         blank=True,
     )
 
+    def create_network_connection(self, destination_port):
+        print("creating connection from "+str(self.id) +
+              " to  "+str(destination_port.id))
+        if (
+            destination_port.network_connection
+            and destination_port.network_connection != self
+        ):
+            from rackcity.views.rackcity_utils import (
+                NetworkConnectionException
+            )
+            raise NetworkConnectionException(
+                "Destination port '" +
+                destination_port.asset.hostname +
+                ":" +
+                destination_port.port_name +
+                "' is already connected to port '" +
+                destination_port.network_connection.asset.hostname +
+                ":" +
+                destination_port.network_connection.port_name +
+                "'. "
+            )
+        if self.network_connection:
+            self.network_connection.network_connection = None
```

Comment:
if this is on network connection create, why do we need to set this to None (wouldn't it default to Null)? I understand why we would need to do this if we're deleting a connection so that the other uni-directional onetoone connection gets deleted too, but why on create?